source/main/index.js

```diff
@@ -257,6 +259,37 @@ const onAppReady = async () => {
       })
   );
 
+  if (process.platform === 'win32') {
+    logger.info('[Custom-Protocol] Set Windows protocol params: ', {
+      execPath: process.execPath,
+      argv: process.argv,
+    });
+    app.setAsDefaultProtocolClient('web+cardano', process.execPath, [
+      process.argv[1],
+    ]);
+  } else {
+    logger.info('[Custom-Protocol] Set Mac / Linux protocol params');
+    app.setAsDefaultProtocolClient('web+cardano');
+    childProcess.exec(
+      'xdg-mime default Daedalus*.desktop x-scheme-handler/web+cardano'
```

Comment:
`xdg-mime default Daedalus*.desktop` must be ran with the right name for the cluster

the install script on linux does that for you, so the JS code doesnt need to run `xdg-mime` on linux